README.md

```diff
@@ -1,11 +1,82 @@
+![Aragon](https://res.cloudinary.com/dacofvu8m/image/upload/v1677353961/Aragon%20CodeArena/osx_blue_logo_lqrvkr.png)
+
 # Aragon JS SDK
 
-This folder contains all the JS packages available on NPM:
+Welcome to the Aragon JS SDK! This SDK allows you to interact with Aragon OSx
+using JavaScript. Whether you are a developer wanting to build on top of Aragon,
+or looking to contribute to the SDK itself, you are in the right place! This
+folder contains all the JS packages available on NPM:
 
 - [Client](./modules/client)
 - [IPFS](./modules/ipfs)
 - [Common](./modules/common)
 
+## Developer Quick Start
+
+The Aragon OSx SDK is Javascript agnostic, which means you can use it with any
+Javascript framework, including popular ones like React, Vite, or Vue. The
+examples in this readme are written using NodeJS
+
+First things first install the client with
+
+```sh
+yarn add @aragon/sdk-client
+```
+
+or
+
+```sh
+npm install @aragon/sdk-client
+```
+
+Then, you'll want to set up the Aragon OSx SDK context within your application
+to have access to the SDK functions.
+
+```typescript
+// 1. Create a context
```

Comment:
We haven't shared what "context" means yet nor where this code should be added. Let's rephrase and say something a bit more explicit like:

In order to set up the configuration for the Aragon SDK, we want to create a "context" variable which will give us access to the web3 provider and signer handling the SDK transactions.

In any file within your project, define and create the Aragon SDK context through something like:
```js
import { Wallet } from 'ethers';

// 1. Define the configuration for the SDK. This will be used to transact using the SDK. 
const minimalContextParams: MinimalContextParams = {
  network: "mainnet", // can also be "goerli", "mumbai", or "polygon"
  web3Providers: "https://rpc.ankr.com/eth", // can be any provider you're familiar with like Infura, Alchemy, Quicknode, etc
  signer: Wallet.createRandom(), // this is who will be signing the transactions. We're using a random wallet in this example for development speed, but feel free to fetch the right wallet from the injected provider in your dApp. Tools like RainbowKit and Wagmi are great here.
};

const context: Context = new Context(minimalContextParams);
```

Some observations on this:
- If we're using `Wallet` from `ethers`, we should add a line importing `ethers` as well - probably when we're installing the SDK into the project.
- Make sure to check that the `graphqlNodes` are not needed for set up. If they're needed under any circumstance, let's chat with OSx to revise the abstraction and make sure it's working as expected.
- Let's check the interfaces and add typing to these variables.